### ISSUE TITLE
Modal windows style diversity when exit from edit properties without save changes

### ISSUE THREAD

**Issue by francoperversi** [geosolutions-it/MapStore2#4537]
### Description
The confirm modal window has different style when exit from the edit properties window without save changes in various situation:

* Cards **Edit properties** button
   - For maps see image 1 in 'Other useful information'
   - For dashboards see image 2 in 'Other useful information'
   - For stories same as dashboards
   - For contexts same as dashboards
   - Editing existing users any confirm modal windows are present
   - Editing existing groups any confirm modal windows are present

* Burger menu **Save/Save as** buttons
   - For maps same as image 2 in 'Other useful information'
   - For dashboards same as maps
   - For stories same as maps
   - For contexts can't text yet
   - Creating new user any confirm modal windows are present
   - Creating new group any confirm modal windows are present

**In conclusion, the propose is the following:**
In all the listed situation, when the user exit from a settings window without saving some changes, the same modal window should appears: image 2 in 'Other useful information'

### Other useful information (optional):
1.
![map_ep](https://user-images.githubusercontent.com/54808782/69360298-969b4600-0c8a-11ea-85d7-ef2074f37515.jpg)
2.
![maps_save](https://user-images.githubusercontent.com/54808782/69360645-36f16a80-0c8b-11ea-9a0f-447f946112c3.jpg)
